### ISSUE TITLE
Documentation: list supported filters in a wiki or in the repository

### ISSUE THREAD

**Issue by adegerard** [chaiNNer-org/chaiNNer#2034]
**Motivation**
Ability to know which filters/functions (nodes) are implemented without installing the software.

**Description**
In a wiki page or as a markdown file in the repository, list filters and options: e.g. blend modes, shapes of the erode/dilate nodes
It might be a very simple list or a more detailed description: link to existing explanation of what this node does and/or document it with a very simple example (like in the OpenCV documentation)

**Comment by joeyballentine** [comment]
I would only do this if we could generate markdown files for it automatically and link them to the wiki page. If we have to do it manually, not a chance that will happen as that will be a nightmare to maintain.

**Comment by RunDevelopment** [comment]
> I would only do this if we could generate markdown files for it automatically and link them to the wiki page. If we have to do it manually, not a chance that will happen as that will be a nightmare to maintain.

Not really. We could automate this. Assuming we host this on a website, we could start the backend fetch `/nodes` as part of the build process of the website. This would give us access to all the info we need, and automated page generation can follow. I imagine a build process where we first fetch `/nodes` and write it into a JSON file, and then build the docs pages from that JSON file.

**Comment by joeyballentine** [comment]
Ah well if the requirement isn't a GitHub wiki but rather just docs available online, I was planning on having the existing docs be built into chaiNNer.app. i have an idea for how to do that that I'd like to try

**Comment by RunDevelopment** [comment]
> Ah well if the requirement isn't a GitHub wiki but rather just docs available online

That's how I interpreted it. We can do more on our website anyway.

**Comment by Yaiol** [comment]
One idea ... one can identify the model using the md5 key of the file.

To deal with the nighmare of unexpilicit name of many models, I renamed all the model files to something more easy to filter like [Architecture]-[Purposes]-[Author]-[Name]-[Scale], I kept original file name into some external sheet and use the md5 as key
When using into Automatic1111 or ComfyUI in can now very easily filters all Anime or Photos models.

Is also how i cleanup my personnal models database with same files coming from different sources with different names.

People accepting the principle could send automatically the information via Chainner

Could be even linked with the new https://openmodeldb.info/ site, no need to have a separate wiki, the new site is just great.
![ChaiNNer Compatibility](https://github.com/chaiNNer-org/chaiNNer/assets/38218161/6b7d6f26-b34d-4998-8045-c8eaf480f7a7)

You could even imagine deeper collaboration between openmodeldb and chaiNNer, user sending some sample of before/after ... maybe i'm seeing too far.

**Comment by joeyballentine** [comment]
That suggestion does not appear to be related to the original post. What exactly are you suggesting we do here?

**Comment by Yaiol** [comment]
My mistake (or my poor english) :) I misread and thought adegard was talking of "models" compatibility.
Well you can still kep the suggestion in mind if you like it.

**Comment by joeyballentine** [comment]
I've always wanted to have some way of downloading models through chaiNNer. If you aren't aware, we actually run OpenModelDB. I did set up a sort of API we'd be able to use for that purpose eventually. If we were to ever do something like that, it would probably just be something like a "Download Model" node where it has a drop-down of compatible models (but even that has its challenges). Anyway, that won't happen for a while regardless.

**Comment by Yaiol** [comment]
Yes I notice you were part of OpenModelDB and you did a great job lately with it, is why I mentionned it actually
Was thinking of linking ChaiNNer with it using md5 of models files and not the name.
Like I want to help OpenModelDB, i download a new model somewhere, is working, ChaiNNer detect the md5 is not part of OpenModelDB, i could do the opposite upload models to the site. Might be a bit complicated but well ... is just a friday idea :)
